apps/toolkit-app/app/core/layouts/Layout.tsx

```diff
@@ -0,0 +1,17 @@
+import Head from "next/head"
+import { FC } from "react"
+
+const Layout: FC<{ title?: string }> = ({ title, children }) => {
```

Comment:
We'll have to add `children` manually as in https://github.com/blitz-js/blitz/pull/3282